### PR TITLE
chore: upgrade deps

### LINKED ISSUE #2376: Please update babel dependencies.
### What problem does this feature solve?
The official [babel 7.0.0](https://github.com/babel/babel/releases/tag/v7.0.0) has been released,  but the version used by `@vue/babel-preset-app` is 'beta.47'.

Thanks for your efforts to create such a great framework.

### What does the proposed API look like?
Update babel dependencies' version from '7.0.0-beta.47' to '7.0.0'.

### PR DESCRIPTION
**Pull request by sodatea**
closes #2376

### CODE REVIEW

File: packages/@vue/eslint-config-standard/package.json [hunk 1]
```diff
@@ -21,9 +21,9 @@
   },
   "homepage": "https://github.com/vuejs/vue-cli/tree/dev/packages/@vue/eslint-config-standard#readme",
   "dependencies": {
-    "eslint-config-standard": "^12.0.0-alpha.0",
-    "eslint-plugin-import": "^2.11.0",
-    "eslint-plugin-node": "^6.0.1",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^7.0.1",
     "eslint-plugin-promise": "^3.7.0",
```
Comment (DRoet) [inline packages/@vue/eslint-config-standard/package.json]: `^4.0.1` ?

File: packages/@vue/eslint-config-standard/package.json [hunk 2]
```diff
@@ -21,9 +21,9 @@
...
     "eslint-plugin-standard": "^3.1.0"
```
Comment (DRoet) [inline packages/@vue/eslint-config-standard/package.json]: `^4.0.0` ?

**Comment by LinusBorg** [comment]
@sodatea babel-plugin-dynamic-import now should support babel 7:

https://github.com/airbnb/babel-plugin-dynamic-import-node/pull/64#issuecomment-428011905

**Comment by Coder-256** [comment]
Don't forget to update this too: https://github.com/vuejs/vue-cli/blob/528c46507a21b76f12bc416d98f9b04c0076cb50/packages/%40vue/eslint-config-typescript/package.json#L25

File: packages/@vue/eslint-config-standard/package.json (hunk 1 above)
Comment (sodatea) [reply packages/@vue/eslint-config-standard/package.json]: I'm planning to open a separate PR to update ESLint to make it easier to review & test.

**Comment by sodatea** [comment]
@andrew-humu See https://github.com/vuejs/vue-cli/projects

**Comment by andrew-humu** [comment]
Thanks for the update, @sodatea !

**Pull request merged**